src/components/bottom_bar/__snapshots__/bottom_bar.test.tsx.snap

```diff
@@ -263,7 +252,7 @@ Array [
   <section
     aria-label="Page level controls"
     class="ouiBottomBar ouiBottomBar--fixed ouiBottomBar--paddingMedium"
-    style="left:12px;right:0;bottom:0"
+    style="left:12px"
```

Comment:
Can we expand the `bottom_bar.test.tsx` test cases to also cover the other positional props (`bottom`, `right`).